packages/models-library/src/models_library/api_schemas_storage.py

```diff
@@ -312,7 +310,7 @@ class FoldersBody(BaseModel):
     destination: dict[str, Any] = Field(default_factory=dict)
     nodes_map: dict[NodeID, NodeID] = Field(default_factory=dict)
 
-    @root_validator()
+    @model_validator(mode="before")
```

Comment:
why before?

Reply:
Right, by default is `after`.

Reply:
but the root_validator was running after in pydantic v1, why now you change it? this will validate too many times and get strings instead of parsed values no?